README.adoc

```diff
@@ -1,3 +1,131 @@
-image:https://travis-ci.org/olaph-io/slack-spring-boot-starter.svg?branch=master["Build Status", link="https://travis-ci.org/olaph-io/slack-spring-boot-starter"]
+:source-highlighter: prettify
+
+:version: 1.0.0.M1
+
+= Slack Spring Boot
+
+Interact with Slack from your Spring Boot application.
+
+image:https://travis-ci.com/kreait/slack-spring-boot-starter.svg?branch=master["Build Status", link="https://travis-ci.org/olaph-io/slack-spring-boot-starter"]
 image:https://sonarcloud.io/api/project_badges/measure?project=io.olaph.slack&metric=alert_status&metric=alert_status[https://sonarcloud.io/dashboard?id=io.olaph.slack]
 image:https://sonarcloud.io/api/project_badges/measure?project=io.olaph.slack&metric=coverage[https://sonarcloud.io/dashboard?id=io.olaph.slack]
+image:https://img.shields.io/badge/version-{version}-fffff.svg[]
+image:https://badges.gitter.im/kreait/slack-spring-boot-starter.svg[https://gitter.im/kreait/slack-spring-boot-starter?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge]
+
+Join the discussion on gitter :)
+
+= Quickstart Guide
+This short guide will guide you through a quick setup
+
+=== Create a Slack App
+
+Go to https://api.slack.com/
+Under "Your Apps" create a new Slack App, give it a name and select the Slack Workspace want for it.
```

Comment:
give it a name and select the slack workspace you want to develop with (you **can't** change this workspace later)